tests/proxyname_test.go

```diff
@@ -5,6 +5,8 @@ import (
 	"errors"
```

Comment:
what about rename the test file as well.. maybe labels_test?

Reply:
yep, will change the name to `label_test`